modules/sre-cluster-access.adoc

```diff
@@ -14,15 +14,19 @@ clusters is controlled through several layers of required authentication, all of
 
-** SRE sends a request tunneled through the Red Hat VPN. This request is made via Corporate Identity and Access Management system (RH IAM); authentication is multi-factor (made up of a password and an ephemeral one-time token). Once the SRE authenticates and is allowed access to the orchestration and management systems, the authorization is managed by Red Hat corporate directory services. The use of RH IAM enables SREs to be managed internally per organization via groups and existing on-boarding/off-boarding processes. Changes to the orchestration and management systems require many layers of approval and are maintained by strict company policy.
+// ** SRE sends a request tunneled through the Red Hat VPN. This request is made via Corporate Identity and Access Management system (RH IAM); authentication is multi-factor (made up of a password and an ephemeral one-time token). Once the SRE authenticates and is allowed access to the orchestration and management systems, the authorization is managed by Red Hat corporate directory services. The use of RH IAM enables SREs to be managed internally per organization via groups and existing on-boarding/off-boarding processes. Changes to the orchestration and management systems require many layers of approval and are maintained by strict company policy.
 
-** Once authorized, SRE logs into the fleet management plane and receives a service account token that the fleet management plane created. The token is valid for twelve minutes. Once the token is no longer valid, it is deleted.
+** SRE connects to the Red Hat VPN. The authentication to the VPN is performed by the Red Hat Corporate Identity and Access Management system (RH IAM). The use of RH IAM enables SREs to be managed internally per organization via groups and existing on-boarding/off-boarding processes, and is multi-factor. Once authenticated and connected, the SRE can access the cloud services fleet management plane. Changes to the cloud services fleet management plane require many layers of approval and are maintained by strict company policy.
+
+
+** Once authorized, SRE logs into the fleet management plane and receives a service account token that the fleet management plane created. The token is valid for fifteen minutes. Once the token is no longer valid, it is deleted.
 
 ** With access granted to the fleet management plane, SRE uses various methods to access clusters, depending on network configuration.
 
-*** Accessing a private or public cluster: Request is sent through a specific Network Load Balancer (NLB) using an encrypted HTTP connection on port 6443. The NLB contains an IP allow-list so the APIs accept connections from a specific set of IPs of which the fleet management plane contains.
+*** Accessing a private or public cluster: Request is sent through a specific Network Load Balancer (NLB) using an encrypted HTTP connection on port 6443.
```

Comment:
```suggestion
*** Accessing a private or public cluster: Request is sent through a specific Network Load Balancer (NLB) by using an encrypted HTTP connection on port 6443.
```
Guidance from IBM Style Word Usage about the term _using_: "To avoid ambiguity, replace this gerund with either _by using_ or _that uses_."